gpytorch/kernels/rbf_kernel_gradgrad.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+import torch
+from linear_operator.operators import KroneckerProductLinearOperator
+
+from .rbf_kernel import RBFKernel, postprocess_rbf
+
+
+class RBFKernelGradGrad(RBFKernel):
+    r"""
+    Computes a covariance matrix of the RBF kernel that models the covariance
+    between the values and first and second (non-mixed) partial derivatives for inputs :math:`\mathbf{x_1}`
+    and :math:`\mathbf{x_2}`.
+
+    See :class:`gpytorch.kernels.Kernel` for descriptions of the lengthscale options.
+
+    .. note::
+
+        This kernel does not have an `outputscale` parameter. To add a scaling parameter,
+        decorate this kernel with a :class:`gpytorch.kernels.ScaleKernel`.
+
+    Args:
```

Comment:
Can you update the doc string to be in the standard sphinx format? (See [the Kernel base class]( https://github.com/cornellius-gp/gpytorch/blob/a2b5fd8c1f164001e8ce018eddc0b86d332cacf6/gpytorch/kernels/kernel.py#L79) for an example)

Reply:
Updated